packages/codebytes/src/MonacoEditor/index.tsx

```diff
@@ -0,0 +1,59 @@
+// DO NOT CHANGE ANYTHING HERE
+// This component is part of the Codebytes MVP and only includes basic configuration around theming
+// We are working on a monaco package in client-modules that has more configuration around themes and languages
+// Monaco as a shared package RFC https://www.notion.so/codecademy/Monaco-editor-as-a-shared-package-1f4484db165b4abc8394c3556451ef6a
+
+import loadable from '@loadable/component';
+import { OnMount } from '@monaco-editor/react';
+import { editor } from 'monaco-editor/esm/vs/editor/editor.api';
+import React, { useCallback, useRef } from 'react';
+import ResizeObserver from 'react-resize-observer';
+
+import { dark } from './theme';
+import { Monaco } from './types';
+
+const ReactMonacoEditor = loadable(() => import('@monaco-editor/react'));
```

Comment:
not sure how this may play with Next.js as it uses [dynamic imports](https://nextjs.org/docs/advanced-features/dynamic-import) instead

Reply:
Best solution here might be to keep this as a normal import, and have the parent component decide to use dynamic imports (next.js) or loadable (monolith) when calling the CodeByteEditor. Updating to this approach for now.